projects/plugins/jetpack/_inc/lib/class-jetpack-podcast-helper.php

```diff
@@ -9,20 +9,46 @@
  * Class Jetpack_Podcast_Helper
  */
 class Jetpack_Podcast_Helper {
+	const MIN_PODCAST_CACHE_TIMEOUT = 30 * MINUTE_IN_SECONDS;
```

Comment:
Since the default now is effectively 12 hours, because that's what the WordPress feed caching uses, why not just use that as the default here?

Ultimately anyone can use the underlying `wp_feed_cache_transient_lifetime` filter and override the minimum anyway.

Reply:
I've updated the minimum to in 1c5d462, though we could also remove this minimum entirely.

Reply:
I've now removed the minimum -- I think users capable of implementing these hooks can do far worse!